src/Orchard.Cms.Web/Modules/Orchard.Setup/Assets/js/strength.js

```diff
@@ -0,0 +1,82 @@
+(function ($) {
+
+    $.fn.strength = function (options) {
+
+        var settings = $.extend({
+            minLength: 8,
+            upperCase: false,
+            lowerCase: false,
+            numbers: false,
+            specialchars: false,
+            target: '',
+            style: ''
+        }, options);
+
+
+        var capitalletters = 0;
+        var lowerletters = 0;
+        var numbers = 0;
+        var specialchars = 0;
+
+        var upperCase = new RegExp('[A-Z]');
+        var lowerCase = new RegExp('[a-z]');
+        var number = new RegExp('[0-9]');
+        var specialchar = new RegExp('([!,%,&,@,#,$,^,*,?,_,~])');
```

Comment:
Thought about it, we could take the same logic as in Identity. Another solution is to use `[~\p{L}]`